identity/app/model/SaveForLaterPageData.scala

```diff
@@ -50,7 +50,8 @@ class SaveForLaterDataBuilder(contentApiClient: ContentApiClient, idUrlBuilder:
         for {
           content <- Some(Content(result))
           faciaContent <- Some(FaciaContentConvert.contentToFaciaContent(result))
-          article <- articles.find( article => article.id == content.metadata.id)
+          // The `replace` is a bit of a hack -- it saves having to touch CAPI or pageId generation
+          article <- articles.find( article => article.id == content.metadata.id.replace("crosswords/", ""))
```

Comment:
I went through a few of our repos to try and find the ultimate source of this bug. I think it lies fairly deep, so thought this would be the least chaotic solution. Let me know if that's not the case